tests/slack_sdk/web/test_internal_utils.py

```diff
@@ -23,44 +24,41 @@ def test_build_unexpected_body_error_message(self):
             """Received a response in a non-JSON format: <!DOCTYPE html><html lang="en"><head><meta charset="utf-8">"""
         )
 
-
-@pytest.mark.parametrize(
```

Comment:
These tests added as part of #1259 actually are not properly working. I've rewritten the tests to work as we expect.